### ISSUE TITLE
Focal loss and box loss = 0.000 when detect text field

### ISSUE THREAD

**Issue by Tien-Thanh** [NVIDIA/retinanet-examples#280]
Hi everyone!
I'm working on detecting MRZ text field on passport card. The width and the height of this field is about 90% width and 15% height of the passport card. I get the focal loss and box loss always equal 0 when training. I think that it's not problem with my dataset and annotation file. Did anyone get this problem?  Could you give me some hints of adjust hyperparameter or something else to address it?
Thanks you!

**Comment by geyyer** [comment]
Hi, try to change anchor-ious argument or anchor size/shape
